.Lib9c.Tests/Action/ArenahelperTest.cs

```diff
@@ -129,10 +129,10 @@ public void ExecuteGetTicketPrice()
                     }
 
                     var arenaInformation = new ArenaInformation(_avatar1Address, roundData.ChampionshipId, roundData.Round);
-                    var max = ArenaHelper.GetMaxPurchasedTicketCount(roundData);
+                    var max = roundData.MaxPurchaseCount;
                     for (var i = 0; i < max; i++)
                     {
-                        arenaInformation.BuyTicket(roundData);
+                        arenaInformation.BuyTicket(roundData.MaxPurchaseCount);
```

Comment:
is this correct change? why not use ArenaHelper?

Reply:
I changed the role of `ArenaHelper.GetMaxPurchasedTicketCount` to `roundData.MaxPurchaseCount`. The intention is correct.

Reply:
it feels strange to test the changed method in this test class. because there is no required `ArenaHelper` anymore, i think this test case freeze with deprecated model for avoid `InvalidBlockStateRootHashException` and create new test in `BattleArenaTest`

Reply:
ok I understand. I will make new unit test.